README.md

```diff
@@ -3,25 +3,72 @@
 :lock: The Bastion
 ==================
 
+## Overview
+
 Bastions are a cluster of machines used as the unique entry point by operational teams (such as sysadmins, developers, database admins, ...) to securely connect to devices (servers, virtual machines, cloud instances, network equipment, ...), usually using `ssh`.
 
-Bastions provides mechanisms for authentication, authorization, traceability and auditability for the whole infrastructure.
+Bastions provide mechanisms for authentication, authorization, traceability and auditability for your whole infrastructure.
+
+Being between your users and your infrastructure, bastions add a layer of abstraction in-between so that your infrastructure doesn't need to know your operational team members individually.
```

Comment:
```suggestion
Being between your users and your infrastructure, The Bastion adds a layer of abstraction in-between so that your infrastructure doesn't need to know your operational team members individually.
```